README.md

```diff
@@ -10,13 +10,14 @@ Vivliostyle Flavored Markdown (VFM), a Markdown syntax optimized for book author
 ### VFM
 
 1. Rule of least surprise
-   - Should be lined and matched to another Markdown syntax.
-1. **Not intended** to be a superset/subset of either CommonMark or GFM.
-1. Backward compatible syntax (should not be incorrectly rendered in Markdown editor like Typora).
+   1. Should be lined and matched to another Markdown syntax.
+   1. Minimize incompatible syntax.
+1. Not necessarily needed to inherit all syntax from CommonMark and GFM.
+   1. Should pick one for its needs, not for its historical reasons.
```

Comment:
In my understanding, this says VFM does not support some kind of annotations on CommonMark and GFM. Is this right? If true, what is removed from CommonMark (or original markdown)? As a specification, it should be clear what annotation is supported (or not supported). I don't think VFM does not support all of unlisted notations in here (because there are still no list notations!).

Note: This is not an objection. I can understand some of notation is ugly (ex. hardline break is very bad to me). But it seems the difference (or removal) is not described and discussed. So it would be better to clear in next discussion, I think.

Reply:
We've been struggling to define a Markdown syntax that has only a clean and concise set of rules but turned out that it costs too much and we have few resources to keep an eye on every syntax.
We may have to rely on existing specs. Then all we have to do is to add/remove syntax from the spec. CommonMark and GFM are promising candidates.

Reply:
Thank you for explanation. I agree with you. It costs too much to make accurate spec. So it would be better to be not sensitive (it's me!).

BTW, I recommend you to use CommonMark for baseline of VFM. I think there are no advantage of GFM. Their extended annotation does not help to make books. For example, its table syntax is poor. It can't combine cells, make a cell having multi-lined text and so on. So I prefer to use customized (extended) CommonMark!